flake.nix

```
@@ -119,7 +126,7 @@
                 # Nix formatter
                 alejandra.defaultPackage.${system}
               ]
-              ++ scriptBins;
+              ++ scriptBins ++ systemSpecificDependencies;
```

I believe there's a shortcut for what you're trying to achieve
```suggestion
              ++ lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks; [CoreFoundation SystemConfiguration]) ++ scriptBins;
```

---

Done in [ccca6cd](https://github.com/osrd-project/osrd/pull/5711/commits/ccca6cd278ac046c3782b7c006a22a5853a7b948).